src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/internalLinks/data.js

```diff
@@ -0,0 +1,610 @@
+module.exports = {
```

Comment:
What's the difference between data.js and fixture.js?

Reply:
Renamed to `inputFixture` and `outputFixture`